.github/workflows/acceptance-public.yml

```diff
@@ -47,15 +47,6 @@ jobs:
       operator_id: ${{ inputs.operator_id }}
       operator_key: ${{ inputs.operator_key }}
 
-  rpc_api_schema_conformity:
```

Comment:
Why remove vs replace?
We don't want to lose the coverage here when running this workflow.

Same note for a few other places you removed it.